challenge-cowsay-two/solution2.js

```diff
@@ -1,18 +1,33 @@
-// =================
-// Stripped down cowsayer CLI, 
-// no libraries or arguments
-// https://nodejs.dev/learn/accept-input-from-the-command-line-in-nodejs
-// =================
+const readline = require("readline");
 
-// 1. Make  a command line interface.
+let rl = readline.createInterface(process.stdin, process.stdout);
 
-// 2. Make supplies for our speech bubble
+let topLine = "_";
+let bottomLine = "-";
+let saying = "";
 
-// 3. Make a cow that takes a string
+const cow = (inPhrase) => {
+  if (!inPhrase) {
+    inPhrase = "nothing to say?";
+  }
+  saying = "      (" + inPhrase + ")";
+  topLine = "       " + topLine.repeat(inPhrase.length);
+  bottomLine = "       " + bottomLine.repeat(inPhrase.length);
 
-const cow = (saying) => {
-    // how did you make the cow before?
-}
+  console.log(topLine);
+  console.log(saying);
+  console.log(bottomLine);
+  console.log("       /");
+  console.log("      /");
+  console.log("^__^ /");
+  console.log("(oo)'_______");
+  console.log("(__)        )-~");
```

Comment:
you can make a function to print out the cow, and we can also use multiple-line printing quotes as well. 
ex-1:  using   template literal    `

 `` line 1

line 2`

ex-2:
" line 1 \n line 2"